tc_hivemind_backend/ingest_qdrant.py

```
@@ -89,7 +89,7 @@ def __init__(
 
         self.clear_cache_after_ingestion = clear_cache_after_ingestion
 
-    def run_pipeline(self, docs: list[Document]) -> list[BaseNode]:
+    def run_pipeline(self, docs: list[Document], transformations: list[TransformComponent] = None) -> list[BaseNode]:
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)